### PR TITLE
feat: create screen

### PR DESCRIPTION
**Pull request by TonNQ**
Create screen

86epuhaam

### CODE REVIEW

File: src/components/Cinema/CreateCinema.tsx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+        provinceCityId: selectedCity ? selectedCity.id : 0
+      }))
+    } else {
+      setNewCinema((prevCinema) => ({
+        ...prevCinema,
+        [name]: value
+      }))
+    }
+
+    setIsFormValid(
```
Comment (ces-anhlam) [inline src/components/Cinema/CreateCinema.tsx:53]: should use react-hook-form for this component. But let's ignore it for now

File: src/components/Cinema/CreateScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+    if (response.status === 200) {
+      toast.success('Create new screen successfully')
+      navigate(path.detailCinema.replace(':id', params.id as string))
+    } else {
+      toast.error('Error occurred while creating a new screen. Try again!')
+    }
+  }
+
+  const handleSaveChanges = async () => {
+    // TODO
```
Comment (ces-anhlam) [inline src/components/Cinema/CreateScreen.tsx:96]: what todo?

**Comment by ces-anhlam** [comment]
@TonNQ Please resolve conflicts

File: src/components/Cinema/CreateScreen.tsx (hunk 2 above)
Comment (TonNQ) [reply src/components/Cinema/CreateScreen.tsx]: I removed it

**Review by ces-anhlam — APPROVED** [review]

**Review by cuongdc03 — APPROVED** [review]

**Pull request merged**